project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject refactor-nrepl "3.0.0-alpha4"
+(defproject refactor-nrepl "3.0.0-alpha10"
```

Comment:
Why the alpha versions keep changing in every commit?

Reply:
> Why the alpha versions keep changing in every commit?

I had shared the alpha with someone and found out that we had a bunch of buggy stuff + buggy Makefile 😞 so I didn't stop until I got something decent shared publicly.

I added a `make install` command in this PR so I won't need alphas as much anyway!